docker-bake.hcl

```diff
@@ -122,6 +123,7 @@ target "encoder" {
 target "encoder-icicle" {
   context    = "."
   dockerfile = "./disperser/cmd/encoder/icicle.Dockerfile"
+  platforms  = ["linux/amd64"]
```

Comment:
```suggestion
  // Currently needed because Dockerfile has amd64 hardcoded in a few places.
  // TODO: make Dockerfile also work for arm.
  platforms  = ["linux/amd64"]
```